system/doc/getting_started/robustness.xml

```diff
@@ -216,7 +216,7 @@ Ping received pong</pre>
       that it does not get killed when it receives abnormal exit
       signals, but all signals will be turned into normal messages on
       the format <c>{'EXIT',FromPID,Reason}</c> and added to the end of
-      the receiving processes message queue. This behaviour is set by:</p>
+      the receiving process's message queue. This behaviour is set by:</p>
```

Comment:
I checked the rules and it depends on the authority. Chicago Manual of Style would say yes, apostrophe only. Others (Oxford University Press, the Modern Language Association, the BBC and The Economist) choose to only use the single possessive apostrophe (`'`) when the word is *plural*. 

In this case, you would write `processes'` but `process's` for the singular form.

I'm personally a fan of Chicago Manual of Style's approach.